.env.docker.example

```
@@ -44,3 +44,10 @@ NEXT_PUBLIC_CLOUDINARY_PRESET_NAME=ci-stub
 
 # === IpInfo (地域判定。任意) ===
 IPINFO_TOKEN=
+
+# === SonarQube (ローカル静的解析。make sonar-scan で使用) ===
+#   1. make sonar-up で起動 → http://localhost:9000 (admin/admin) でログイン
+#   2. My Account → Security でトークンを発行し SONAR_TOKEN に設定
+#   SONAR_HOST_URL はローカル scanner では compose 内部名で解決するため通常変更不要。
+SONAR_TOKEN=
+SONAR_HOST_URL=http://localhost:9000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SONAR_HOST_URL key should go before the SONAR_TOKEN key

(UnorderedKey)